### PR TITLE
feat: command to view relationship by model

### LINKED ISSUE #9844: [mesheryctl] New command `exp relationship`
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Desired Behavior

Create an `exp relationship [list|view|search]` command similar to that of `exp model command.`
- [ ] Implement command to List Relationships
- [x] Implement command to  View Relationships
- [ ] Implement command to Search Relationships
- [ ] Develop test cases to ensure command functionality.

---

### Contributor Resources

- ⌨️ [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](http://discuss.meshery.io) and [Layer5 Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by RipulHandoo**
**Notes for Reviewers**

This PR fixes #9844 

1. Sub-Command `[view]` for relationship.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
Nice 👌🏻

**Comment by alphaX86** [comment]
This PR can be linked to this issue #9844

**Review by alphaX86 — COMMENTED** [review]
@MUzairS15 are we planning this command to be within `system` command itself. Or as a separate one?

File: mesheryctl/internal/cli/root/system/relationship.go [hunk 1]
```diff
@@ -0,0 +1,261 @@
+// Copyright 2023 Layer5, Inc.
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/system/relationship.go:1]:
```suggestion
// Copyright Meshery Authors
```

File: mesheryctl/internal/cli/root/system/relationship.go [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+		err = ctx.ValidateVersion()
+		if err != nil {
+			return err
+		}
+		return nil
+	},
+	Args: func(_ *cobra.Command, args []string) error {
+		const errMsg = "Usage: mesheryctl exp relationships view [model-name]\nRun 'mesheryctl exp relationships view --help' to see detailed help message"
+		if len(args) == 0 {
+			return fmt.Errorf("model name isn't specified\n\n%v", errMsg)
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/system/relationship.go:70]: Meshkit error

File: mesheryctl/internal/cli/root/system/relationship.go [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+		} else {
+			selectedModel = selectRelationshipPrompt(relationshipsResponse.Relationships)
+		}
+
+		var output []byte
+
+		// user may pass flag in lower or upper case but we have to keep it lower
+		// in order to make it consistent while checking output format
+		outFormatFlag = strings.ToLower(outFormatFlag)
+		if outFormatFlag == "yaml" {
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/system/relationship.go:130]: What if the user provides "yml"?

Comment (MUzairS15) [reply mesheryctl/internal/cli/root/system/relationship.go:130]: The default format is yaml/yml, correct?

File: mesheryctl/internal/cli/root/system/relationship.go [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+		return relationshipArray[i]
+	}
+}
+
+var RelationshipCmd = &cobra.Command{
+	Use:   "relationships",
+	Short: "View list of relationships and detailed information of a specific relationship",
+	Long:  "View list of relationships and detailed information of a specific relationship",
+	Example: `
+// To view list of components
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/system/relationship.go:178]: Update the docs and usages.

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@RipulHandoo when will this be finished?

**Comment by leecalcote** [comment]
Why is this being closed?

**Pull request closed without merging**